aiosasl/stringprep.py

```diff
@@ -33,6 +33,7 @@
 
 import stringprep
 import unicodedata
+unicodedata = unicodedata.ucd_3_2_0
```

Comment:
Why not ``from unicodedata import ucd_3_2_0 as unicodedata`` if we have to do aliases?

Reply:
No comment.